package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/theme",
-  "version": "4.15.4",
+  "version": "4.15.5",
```

Comment:
Same comment as the other release PR about updating the `package-lock.json` file.

Reply:
Good catch, got it.